### PR TITLE
Add tooltip and legends for Hiks-Dufour and Siegfried Metadata Shop layers

### PR DESCRIPTION
**Pull request by Ariadnigaki**
Test link : https://mf-geoadmin3.dev.bgdi.ch/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fltgar__layers

### CODE REVIEW

**Comment by p1d1d1** [comment]
@Ariadnigaki Test link doesn't seem to work.

**Comment by Ariadnigaki** [comment]
Test link: https://mf-geoadmin3.dev.bgdi.ch/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fltgar__layers&lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.hiks-siegfried-ta50.metadata,ch.swisstopo.hiks-siegfried-ta25.metadata,ch.swisstopo.hiks-dufour.metadata&layers_visibility=false,false,true

**Comment by p1d1d1** [comment]
@Ariadnigaki nice job. Many thanks

**Comment by gjn** [comment]
Good stuff. @p1d1d1 is this validated with Datenherr for the 26th?

**Comment by p1d1d1** [comment]
@gjn "Procotols" are signed.
Layers should be in "int" on 19.10 and in "prod" on 26.10. 
SHOP deploy in parallel.

**Comment by gjn** [comment]
Great. Thanks alot for the info.

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -2441,3 +2441,39 @@ class Lotabweichungen(Base, Vector):
...
 
 register('ch.swisstopo.lotabweichungen', Lotabweichungen)
+
+
+class HiksDufourMetadata(Base, ShopStandardClass, Vector):
+    __tablename__ = 'view_gridstand_dufour_shop'
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __bodId__ = 'ch.swisstopo.hiks-dufour.metadata'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    number = Column('s_map_number', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: Unicode or Float/Integer if really a number

File: chsdi/models/vector/stopo.py [hunk 2]
```diff
@@ -2441,3 +2441,39 @@ class Lotabweichungen(Base, Vector):
...
+
+register('ch.swisstopo.hiks-dufour.metadata', HiksDufourMetadata)
+
+
+class HiksSiegfriedTa25Metadata(Base, ShopStandardClass, Vector):
+    __tablename__ = 'view_gridstand_siegfried_ta25_shop'
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __bodId__ = 'ch.swisstopo.hiks-siegfried-ta25.metadata'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    number = Column('s_map_number', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: same here

File: chsdi/models/vector/stopo.py [hunk 3]
```diff
@@ -2441,3 +2441,39 @@ class Lotabweichungen(Base, Vector):
...
+
+register('ch.swisstopo.hiks-siegfried-ta25.metadata', HiksSiegfriedTa25Metadata)
+
+
+class HiksSiegfriedTa50Metadata(Base, ShopStandardClass, Vector):
+    __tablename__ = 'view_gridstand_siegfried_ta50_shop'
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __bodId__ = 'ch.swisstopo.hiks-siegfried-ta50.metadata'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    number = Column('s_map_number', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: same here

File: chsdi/templates/htmlpopup/hiks_dufour_metadata.mako [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <tr><td class="cell-left">${_('ch.swisstopo.hiks-dufour.metadata.s_map_number')}</td>         <td>${c['attributes']['s_map_number'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.swisstopo.hiks-dufour.metadata.s_title_de')}</td>           <td>${c['attributes']['s_title_de'] or '-'}</td></tr>
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/hiks_dufour_metadata.mako]: we only have title de? otherwise it should be language specific

**Comment by loicgasser** [comment]
We have 3 times the same mako template and a lot of duplicated translations.
If this layer is not searchable, you don't need to follow `layerName.attribute` convention and you could use a single template. TO BE added to the doc.
For the translations you may already find the one you need and therefore you probably don't need to add new ones. TO be checked.
Thanks a lot for this.

File: chsdi/templates/htmlpopup/hiks_dufour_metadata.mako (hunk 4 above)
Comment (Ariadnigaki) [inline chsdi/templates/htmlpopup/hiks_dufour_metadata.mako]: No there are titles for all the languages. However, the title is the same for all of them. For this reason i only used s_title_de. So, do i have to add s_title_fr, s_title_it, etc for all the languages even if they are the same?

Comment (loicgasser) [inline chsdi/templates/htmlpopup/hiks_dufour_metadata.mako]: I understand, but I would say yes, this way we don't need to go back in the code if these titles are updated.

Comment (Ariadnigaki) [inline chsdi/templates/htmlpopup/hiks_dufour_metadata.mako]: Ok, thank you for the feedback! I will make the proposed changes.

**Comment by loicgasser** [comment]
I rebased, good changes, appriciated. ;) thanks @pfanguin waiting for Jenkins and I merge

**Pull request merged**